Mathlib/AlgebraicGeometry/GroupScheme/HopfAffine.lean

```diff
@@ -0,0 +1,127 @@
+/-
+Copyright (c) 2025 Yaël Dillies, Christian Merten, Michał Mrugała, Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies, Christian Merten, Michał Mrugała, Andrew Yang
+-/
+import Mathlib.AlgebraicGeometry.Pullbacks
+import Toric.GroupScheme.SchemeOver
+import Toric.Mathlib.Algebra.Category.CommAlg.Basic
+import Toric.Mathlib.CategoryTheory.Limits.Preserves.Shapes.Over
+import Toric.Mathlib.CategoryTheory.Monoidal.Grp_
```

Comment:
?!

Reply:
It's WIP! Just a placeholder for the nice result to get the nice number